### PR TITLE
Add fixes to PayPal process

### PR DESCRIPTION
**Pull request by tatsta**

### CODE REVIEW

File: tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php [hunk 1]
```diff
@@ -23,15 +24,12 @@ public function performPaymentMethodActionsOutsideShop() : void
         $this->performPaypalLogin();
 
         try {
-            $this->preparedClick($this->getLocator()->continue, 80);
-        } catch (WebDriverException $e) {
-            //sometimes we need to accept cookies first
-            $this->waitForText($this->getLocator()->payment_page_text, 60);
-            $this->preparedClick($this->getLocator()->accept_cookies, 80);
-            $this->preparedClick($this->getLocator()->continue, 80);
+            $this->preparedClick($this->getLocator()->pay_now_start, 60);
```
Comment (tomazpu) [inline tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php:27]: In the future we need to think of a helper loading the JSON locator file, so we don't direct access($this->getLocator()->pay_now_start) the values. Then we can introduce validation on the files also.

Comment (tomazpu) [inline tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php:27]: In this method, I see in line 22 is a return type void, if then please add them in all methods or remove this one.

File: tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php [hunk 2]
```diff
@@ -52,7 +50,26 @@ public function performPaypalLogin()
     }
 
     // we need to define this method for consistency, because it will be called in every scenario, empty method just means do nothing here
+
+    /**
+     * @return mixed|void
+     */
     public function fillFieldsInTheShop()
```
Comment (tomazpu) [inline tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php:57]: I suggest splitting the iPreformPayment interface into two interfaces, as now there are only two classes implementing it(less work to refactor). Then you don't need the method where there are no fillFields, as there will be some other payment method down the road with no fields.
`
interface iPreformPayment
interface iPreformFillFiels

class PayPalStep extends GenericPaymentMethodStep implements iPerformPayment
class CreditCardStep extends GenericPaymentMethodStep implements iPerformPayment, iPreformFillFields
`

File: tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php (hunk 1 above)
Comment (tatsta) [reply tests/_support/Step/Acceptance/PaymentMethod/PayPalStep.php:27]: This this is done already. getLocator returns an Object of class locator, that loads json file in the initialisation phase

**Pull request merged**